src/main/java/Duke/IndexOutOfRange.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class IndexOutOfRange extends DukeException {
```

Comment:
Should this exception name follow the existing naming pattern you've established i.e. [description]Exception?

```suggestion
public class IndexOutOfRangeException extends DukeException {
```